docs/snippets/common/msw-addon-initialize.ts.mdx

```diff
@@ -14,15 +14,8 @@ import { initialize, mswDecorator } from 'msw-storybook-addon';
 initialize();
```

Comment:
I think if you add the link to the docs in one place, it should be in all places:

```
/*
 * Initializes MSW
 * See https://github.com/mswjs/msw-storybook-addon#configuring-msw
 * to learn how to customize it
 */
```

Reply:
Are you suggesting we add a similar comment above other usages of MSW?